lib/rules/no-global-variables.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+/**
+ * @fileoverview Disallow variables within the global (script) scope
+ * @author Vadim Ruban
+ */
+
+module.exports = function (context) {
+  var hasTestSuite = false
+  var globalVariableDeclarationNodes = []
+
+  return {
+    'Program > VariableDeclaration': function (node) {
+      var init = node.declarations[0].init
+
+      switch (init.type) {
+        case 'CallExpression': {
+          if (init.callee.name !== 'require') {
+            globalVariableDeclarationNodes.push(node)
+          }
+          break
+        }
+        case 'ArrowFunctionExpression':
```

Comment:
what about
```js
const getAthing => 'aThing'
```
And then calling `getAthing()`  inside a test ? Does that need to be a warning?